net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/forms/dialogEditWireguardClient.xml

```diff
@@ -0,0 +1,40 @@
+<form>
+    <field>
+        <id>client.enabled</id>
+        <label>Enabled</label>
+        <type>checkbox</type>
+        <help>This will enable or disable the client config.</help>
+    </field>
+    <field>
+        <id>client.name</id>
+        <label>Name</label>
+        <type>text</type>
+        <help>Set the name for this instance.</help>
+    </field>
+    <field>
+        <id>client.pubkey</id>
+        <label>Public Key</label>
+        <type>text</type>
+        <help>Public key of this instance.</help>
+    </field>
+    <field>
+        <id>client.tunneladdress</id>
+        <label>Tunnel Address</label>
+        <style>tokenize</style>
+        <type>select_multiple</type>
+        <allownew>true</allownew>
+        <help>List of addresses to configure on the tunnel adapter. Please use CIDR notation like 10.0.0.1/24.</help>
+    </field>
+    <field>
+        <id>client.serveraddress</id>
+        <label>Endpoint Address</label>
+        <type>text</type>
+        <help>Set public IP address the endpoint listens to.</help>
```

Comment:
Endpoints don't listen. This doesn't make sense.

Reply:
If I put it in one field (IP and Port) then I cannont use the validations inherited by FieldType, in config templating they are glued together again. It's just a validation against user errors